worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/ChunkStreamManager.java

```diff
@@ -44,9 +44,41 @@ public class ChunkStreamManager {
   private final AtomicLong nextStreamId;
   // StreamId -> StreamState
   protected final ConcurrentHashMap<Long, StreamState> streams;
+  // StreamId -> StreamRegisterState
+  protected final ConcurrentHashMap<Long, StreamRegisterState> streamRegisterStates;
```

Comment:
Just recording non-range streams could be enough. If there is non range read stream, the origin shuffle file should not be deleted.